src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml

```diff
@@ -36,7 +36,18 @@
                     {
                       <span class="govuk-error-message" asp-validation-for="GivenName"></span>
                     }
-                    <input class="govuk-input govuk-!-width-two-thirds" id="GivenName" name="GivenName" type="text" asp-for="GivenName" sfa-validation-for="GivenName" sfa-validationerror-class="govuk-input--error" autocomplete="given-name" spellcheck="false" />
+                    <input 
+                        class="govuk-input govuk-!-width-two-thirds"
+                        id="GivenName"
+                        name="GivenName"
+                        type="text"
+                        asp-for="GivenName"
+                        sfa-validation-for="GivenName"
+                        sfa-validationerror-class="govuk-input--error"
+                        autocomplete="given-name"
+                        spellcheck="false"
+                        maxlength="10000"
```

Comment:
Perhaps this should be the same length as most Given names in the Database e.g. 250

Reply:
The reason for increase was to match 10000 as that is the character limit in the WAF;